extensions/wrapper/wrapper-common-api/src/main/java/de/sovity/edc/ext/wrapper/api/common/model/UiAsset.java

```diff
@@ -56,6 +56,9 @@ public class UiAsset {
     @Schema(description = "Asset Description", requiredMode = Schema.RequiredMode.NOT_REQUIRED)
     private String description;
 
+    @Schema(description = "Asset Description Short Text", requiredMode = Schema.RequiredMode.NOT_REQUIRED)
```

Comment:
Let's add some documentation to both description and shortDescription:

- Description: Let's add "Supports markdown."
- Short Description: Let's add "Generated from description, contains no markdown"